include/mbgl/ios/MGLMapboxEvents.h

```diff
@@ -58,4 +54,7 @@ extern NSString *const MGLEventGestureRotateStart;
 //
 + (void) flush;
 
+// Main thread only
++ (void)validate;
+
 @end
```

Comment:
Why is this header public at all? The user never needs to call it and it's not brought into `MapboxGL.h`.